xcixor/hms
# 160411331 add employee

### Purpose
- The purpose of this PR is to introduce employee creation
### How to test
- Clone the repo
- Install the dependecies (all)
- Run npm start
- Proceed to the HR dashboard by clicking the login button (no credentials)
- Click on add employee on the left side menu
- A form appears, Include the necessary fields
- Click save
- You should get a pop message notifying you of the successful employee creation
